GitOps.md

```diff
@@ -0,0 +1,17 @@
+---
+title: GitOps
+status: Feedback Appreciated
+category: concept
+---
+
+
+## What it is
+
+GitOps is a [set of principles](https://github.com/open-gitops/documents/blob/main/PRINCIPLES.md) for operating and managing software systems. In GitOps, environment specifics are defined, or "declared," in a repository with the application code — the system of record.
```

Comment:
Good but we need a few more lines to describe what it is beyond principles. GitOps is also a set of practices that involve applying that stored state in a consistent way. I appreciate that we're attempting to weave in the work produced by the gitops working group but their definition is tailored to an audience of existing gitops practitioners. Our definition needs to be more expansive.